.asf.yaml

```diff
@@ -0,0 +1,17 @@
+github:
```

Comment:
We should really setup the notifications as well, so those are going out to the mailing lists.

Reply:
as of now it's already I place... but you might be right... this file may override current behavior